docs/Account/methods.md

```diff
@@ -238,8 +239,41 @@ const { transactionHash, userOperationReceipt } = await wait();
 
 **Returns**
 
-- userOpResponse (`Promise<UserOpResponse>`): userOpResponse that you can use to track user operation.
+- userOpResponse (`Promise<UserOpResponse>`): The method returns an object of type `UserOpResponse` which has a `userOpHash` and two methods: `wait()` and `waitForTxHash()`.
 
+  ```ts
+  type UserOpResponse = {
+    userOpHash: string;
+    wait(_confirmations?: number): Promise<UserOpReceipt>;
+    waitForTxHash(): Promise<UserOpStatus>;
+  };
+
+  type UserOpReceipt = {
+    userOpHash: string;
+    entryPoint: string;
+    paymaster: string;
+    actualGasCost: Hex;
+    actualGasUsed: Hex;
+    success: "true" | "false"; // check for this flag to be true for successful call data execution
+    reason: string;
+    logs: Array<any>;
+    receipt: any;
+  };
+  ```
+
+  The `wait()` method resolves when the user operation is dispatched by the bundler on-chain and gets mined. The `waitForTxHash()` method returns a `UserOpStatus` object which includes the transaction hash and the receipt once added on-chain.
+  ```ts
+    const { transactionHash } = await userOpResponse.waitForTxHash();
+    console.log("transaction Hash", transactionHash);
+
+    const userOpReceipt  = await userOpResponse.wait();
+    
+    if(userOpReceipt.success == 'true') { // indicates that the transaction was successful without any revert
```

Comment:
Would be better to use the term "user operation" instead of transaction here.

--> indicates that the user operation was successful without any revert